plugins/filter/json_to_xml.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+"""
+The index_of filter plugin
+"""
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+    name: json_to_xml
+    author: Ashwini Mhatre (@amhatre)
+    version_added: "1.0.0"
+    short_description: convert given json string to xml
+    description:
+        - This plugin converts the xml string to json.
+        - Using the parameters below- C(data|ansible.utils.json_to_xml)
+    options:
```

Comment:
The `engine` option is not added as mentioned in the issue description. We want to support passing the conversion library to use within the filter plugin. SO that if required new library support can be added.